docs/integrate-key-server.md

```diff
@@ -1,7 +1,7 @@
 
 ## Remote attestation and key management (manual)
 
-By default, the Trusted Certificate Service (TCS) expects the CA private key and root certificate are provisioned by an external key management services like [Intel® KMRA](https://01.org/key-management-reference-application-kmra). For integrating such key servers the SGX operator provides a Kubernetes CRD API named `QuoteAttestation`, which is based on [SGX ECDSA attestation](https://www.intel.com/content/www/us/en/developer/articles/technical/quote-verification-attestation-with-intel-sgx-dcap.html) defined by the [Intel® SGX Data Center Attestation Primitives](https://github.com/intel/SGXDataCenterAttestationPrimitives).
+By default, the Trusted Certificate Service (TCS) expects the CA private key and root certificate are provisioned by an external key management service like [Intel® KMRA](https://01.org/key-management-reference-application-kmra). For key server integration the TCS provides a Kubernetes CRD API named `QuoteAttestation`, which is based on [SGX ECDSA attestation](https://www.intel.com/content/www/us/en/developer/articles/technical/quote-verification-attestation-with-intel-sgx-dcap.html) defined by the [Intel® SGX Data Center Attestation Primitives](https://github.com/intel/SGXDataCenterAttestationPrimitives).
```

Comment:
Currently, the default behavior is self-sign key and certificate, Quote attestation is only when `tcsissuer.spec.selfSign` set to `false`.